src/skydb_v2_testall.test.js

```diff
@@ -0,0 +1,654 @@
+const axios = require("axios");
+const MockAdapter = require("axios-mock-adapter");
+
+const { SkynetClient } = require("../index");
+
+const {
+  stringToUint8ArrayUtf8,
+  getSkylinkUrlForPortal,
+  MAX_REVISION,
+  DEFAULT_SKYNET_PORTAL_URL,
+  URI_SKYNET_PREFIX,
+  getEntryUrlForPortal,
+  DELETION_ENTRY_DATA,
+  MAX_ENTRY_LENGTH,
+  JsonData,
+  JSONResponse,
+} = require("skynet-js");
+
+const { getSettledValues } = require("./utils_testing");
+const { checkCachedDataLink } = require("./skydb_v2");
+const { toHexString } = require("./utils_string");
+const { decodeSkylink } = require("./encoding");
+
+// Generated with genKeyPairFromSeed("insecure test seed")
+const [publicKey, privateKey] = [
+  "658b900df55e983ce85f3f9fb2a088d568ab514e7bbda51cfbfb16ea945378d9",
+  "7caffac49ac914a541b28723f11776d36ce81e7b9b0c96ccacd1302db429c79c658b900df55e983ce85f3f9fb2a088d568ab514e7bbda51cfbfb16ea945378d9",
+];
+
+const dataKey = "app";
+const skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg";
+const sialink = `${URI_SKYNET_PREFIX}${skylink}`;
+const jsonData = { data: "thisistext" };
+const fullJsonData = { _data: jsonData, _v: 2 };
+const legacyJsonData = jsonData;
+const merkleroot = "QAf9Q7dBSbMarLvyeE6HTQmwhr7RX9VMrP9xIMzpU3I";
+const bitfield = 2048;
+
+const portalUrl = DEFAULT_SKYNET_PORTAL_URL;
+const client = new SkynetClient(portalUrl);
+const uploadUrl = `${portalUrl}/skynet/skyfile`;
+const skylinkUrl = getSkylinkUrlForPortal(portalUrl, skylink);
+const registryPostUrl = `${portalUrl}/skynet/registry`;
+const registryGetUrl = getEntryUrlForPortal(portalUrl, publicKey, dataKey);
+
+// Hex-encoded skylink.
+const data = "43414241425f31447430464a73787173755f4a34546f644e4362434776744666315579735f3345677a4f6c546367";
+const revision = "11";
+const entryData = {
+  data: data,
+  revision: revision,
+  signature:
+    "33d14d2889cb292142614da0e0ff13a205c4867961276001471d13b779fc9032568ddd292d9e0dff69d7b1f28be07972cc9d86da3cecf3adecb6f9b7311af809",
+};
+
+const newTimeout = 60000;
+jest.setTimeout(newTimeout);
+jest.useRealTimers();
```

Comment:
Remove.

Reply:
hello m-cat,

Here I should only remove lines 65-58.
But I shouldn't delete the file? Or?

Reply:
Oh yeah, I meant to ask why we have this file. It seems like it repeats tests from `skydb_v2.test.js`. Sorry if I missed an explanation somewhere.

Reply:
Hello m-cat,

What google translate translates is a little weird, so I hope I got it right.

I haven't done everything yet, but I'll push my current status right away.